corehq/apps/translations/app_translations/download.py

```diff
@@ -19,39 +19,60 @@
     get_module_sheet_name,
     get_modules_and_forms_row,
 )
+from corehq.apps.translations.generators import EligibleForTransifexChecker
 
 
-def get_bulk_app_single_sheet_by_name(app, lang):
+def get_bulk_app_single_sheet_by_name(app, lang, eligible_for_transifex_only=False):
+    checker = EligibleForTransifexChecker(app)
+
     rows = []
     for module in app.modules:
+        if eligible_for_transifex_only and checker.exclude_module(module):
+            continue
         sheet_name = get_module_sheet_name(module)
         rows.append(get_name_menu_media_row(module, sheet_name, lang))
         for module_row in get_module_rows([lang], module):
+            if eligible_for_transifex_only:
+                field_name, field_type, translation = module_row
```

Comment:
add the other place where we do something similar,
https://github.com/dimagi/commcare-hq/pull/24604/files#diff-27f0b75dc11241833e40eb967bbb737aR91
we are assuming more than 3 entries in `module_row`, fine to ignore that here?

Reply:
There are only ever three values in `module_row` for single-sheet uploads because there is only one language.